task.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task 
+mkdir -p task
```

Comment:
При использовании флага `-p` можно сразу так делать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```